src/isar/config/settings.py

```diff
@@ -179,6 +186,25 @@ class Settings(BaseSettings):
     # Name or unique ID of robot
     ROBOT_ID: str = Field(default="R2-D2")
```

Comment:
Move this to RobotSettings and change default value to "default-robot"?

Reply:
I started working on it but ended up deciding it is the next pull request. Mainly because it required a lot of change in Flotilla and I don't want this PR to grow any further.

Reply:
Yeah, I agreed